apitools/base/protorpclite/messages.py

```diff
@@ -250,8 +251,7 @@ class MyMessage(Message):
         outer_definition_name = cls.outer_definition_name()
         if outer_definition_name is None:
             return six.text_type(cls.__name__)
-        else:
-            return u'%s.%s' % (outer_definition_name, cls.__name__)
+        return u'%s.%s' % (outer_definition_name, cls.__name__)
```

Comment:
It is subjective, but having return at the end irrespective of condition is less error prone, eg if/elif/elif/else each doing return, if you forget else then it would silently return nothing.